.circleci/config.yml

```diff
@@ -223,7 +219,7 @@ workflows:
             branches:
```

Comment:
upload_production is missing a `requires` but I'm assuming it gets kicked off when the merge to master happens because it's `only: - master`

Reply:
That's how it was previously; I think it's fine because the upload stage does a build.